torch/csrc/generic/methods/TensorCompare.cwrap

```diff
@@ -99,22 +94,11 @@
 ]]
 
 [[
-  name: gt
-  only_stateless: True
+  name: gt_t
```

Comment:
I agree, a `dtype` kwarg would be very useful in a number of cases.
Currently, if we want to generate say a long tensor of consecutive integers, we need to pass a `LongTensor` in the `out` kwarg of the function, which is a bit annoying.